docker/compose/aptos-node/docker-compose.yaml

```diff
@@ -22,8 +22,9 @@ services:
     ports:
       - "6180:6180"
       - "6181:6181"
-      - "80:80"
+      - "80:8180"
```

Comment:
Why do we change this?

Reply:
Because haproxy is listening to 8180m and no one is listening on 80, and as long as we provide the option to use this this should work.